ansible/roles/apigateway/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
-# This role will install controller in group 'controllers' in the environment inventory
-# In deploy mode it will deploy controllers.
-# In clean mode it will remove the controller containers.
+# This role will install apigateway in the group 'apigateway' in the environment inventory
```

Comment:
Comment nit.  I think the role does not control the hosts on which the role is being applied.